src/Sulu/Bundle/CategoryBundle/Infrastructure/Sulu/Search/CategoryReindexProvider.php

```diff
@@ -96,14 +97,24 @@ private function loadCategories(array $identifiers = []): iterable
             $parameters = [];
 
             foreach ($identifiers as $index => $identifier) {
-                $id = \explode('::', $identifier)[1];
-                $locale = \explode('::', $identifier)[2];
+                $resourceKey = \explode('::', $identifier)[0];
+
+                if (CategoryInterface::RESOURCE_KEY !== $resourceKey) {
```

Comment:
@alexander-schranz lets discuss sometimes why this reindexer is called when an account is saved 🤔